### PR TITLE
feat: get deployments for the email

### PR DESCRIPTION
**Pull request by YounixM**

### CODE REVIEW

File: app/login/login.styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  max-height: 320px;
+  overflow: auto;
+}
+
+.error-container {
+  @apply border-signoz_amber-400/20 bg-signoz_amber-400/10 text-signoz_amber-400;
+}
+
+.btn-secondary {
+  background: rgba(171, 189, 255, 0.08);
```
Comment (YounixM) [inline app/login/login.styles.css:15]: We don't have color variables that handle opacity. Hence the direct rgba

File: app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+
+  const handleNoDeployments = () => {
+    setNoDeployments(true)
+  }
+
+  const handleLogin = async () => {
+    setIsSubmitting(true)
+    setSubmitFailed(false)
+
+    const verifyWorkSpaceSetupURL = `${BASE_URL}/deployments/esearch?q=${workEmail}`
```
Comment (therealpandey) [inline app/login/page.tsx]: All query-parameters should be url encoded @YounixM.

**Review by therealpandey — CHANGES_REQUESTED** [review]

File: app/login/page.tsx (hunk 2 above)
Comment (YounixM) [reply app/login/page.tsx]: Done ✅

**Review by vikrantgupta25 — APPROVED** [review]

File: app/login/page.tsx [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+    setWorkEmail('')
+    setSubmitSuccess(false)
+    setUserTenants([])
+    setIsSubmitting(false)
+    setNoDeployments(false)
+    setErrors({})
+  }
+
+  const handleGetStarted = () => {
+    router.push(`/teams?q=${workEmail}`)
```
Comment (vikrantgupta25) [inline app/login/page.tsx]: encode this as well

**Review by vikrantgupta25 — CHANGES_REQUESTED** [review]
encode this as well

**Review by therealpandey — APPROVED** [review]

File: app/login/page.tsx (hunk 3 above)
Comment (YounixM) [reply app/login/page.tsx]: done ✅

**Review by vikrantgupta25 — APPROVED** [review]

**Pull request merged**